### PR TITLE
Add "d" suffix to libraries in Debug configuration

### PR DESCRIPTION
**Pull request by david-antiteum**
Closes case #109

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -213,6 +213,7 @@ if(NOT QTKEYCHAIN_STATIC)
 else()
     add_library(${QTKEYCHAIN_TARGET_NAME} STATIC ${qtkeychain_SOURCES} ${qtkeychain_MOC_OUTFILES} ${qtkeychain_QM_FILES})
 endif()
+set_target_properties( ${QTKEYCHAIN_TARGET_NAME} PROPERTIES DEBUG_POSTFIX "d" )
```
Comment (frankosterfeld) [inline CMakeLists.txt:216]: I think this is only common on Windows, so this should probably only be applied there

Comment (david-antiteum) [reply CMakeLists.txt]: New commit, just for WIndows

Comment (david-antiteum) [reply CMakeLists.txt]: Ejem.. wrong commit message ... should be: Only set suffix in Windows

**Comment by frankosterfeld** [comment]
Thanks!

**Pull request merged**